scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -675,4 +725,19 @@ public void onSuccess(Long result) {
         });
     }
 
+    private Set<String> parseSignals(String result) {
+        Set<String> signals = new HashSet<>();
+        JSONValue jsonValue = null;
+        try {
+            jsonValue = SchedulerJSONUtils.parseJSON(result);
+        } catch (org.ow2.proactive_grid_cloud_portal.common.client.json.JSONException e) {
```

Comment:
why we have all path of the exception here? org.ow2.proactive_grid_cloud_portal.common.client.json.JSONException

Reply:
I changed it and used the import